PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -76,6 +77,8 @@
 run2_jme_2016.toModify( tightJetIdLepVeto.filterParams, version = "WINTER16" )
 run2_jme_2017.toModify( tightJetId.filterParams, version = "WINTER17" )
 run2_jme_2017.toModify( tightJetIdLepVeto.filterParams, version = "WINTER17" )
+run2_miniAOD_devel.toModify( tightJetId.filterParams, version = "RUN2ULCHS" )
+run2_miniAOD_devel.toModify( tightJetIdLepVeto.filterParams, version = "RUN2ULCHS" )
```

Comment:
The case of run2_nanoAOD_106Xv1 is left unspecified and the 2016-UL-V1, 2017-UL-V1 will get the EOYjetID.

should be like this

for modifier in run2_nanoAOD_106Xv1, run2_miniAOD_devel: 
    modifier.toModify( tightJetId.filterParams, version = "RUN2ULCHS" )
    ....